common/include/opae/cxx/handle.h

```diff
@@ -36,14 +36,26 @@ namespace opae {
 namespace fpga {
 namespace types {
 
+/** An allocated accelerator
```

Comment:
Should we use `accelerator` or something more generic like `fpga resource`?
A `handle` can represent either logical accelerator object or a physical fpga object.
In other words, can a fpgaconf be rewritten to use this C++ API (upper layer included) and if so, what would it look like?

Reply:
Hmm.. I was purposefully staying away from fpga so that we didn't limit ourselves.  How about "accelerator resource"?

Reply:
Limit ourselves to just FPGA SW? The namespace already includes fpga...